src/Generator/Generator.csproj

```diff
@@ -50,6 +50,7 @@
     <Using Include="System.Linq" />
     <Using Include="System.Text" />
     <Using Include="System.Text.RegularExpressions" />
+	<Using Include="System.Diagnostics.CodeAnalysis"/>  
```

Comment:
Looks like this was a tab instead of spaces.

Reply:
Updated.